src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+      case 'removeProperties':
+        action.keysToRemove.forEach(key => delete state[key]);
+        break;
+      case 'clear':
+        if (Object.keys(state).length > 0) {
+          Object.keys(state).forEach(key => delete state[key]);
+        }
+        break;
+      default:
+        return state;
```

Comment:
In the 'default' case of switch, it is not necessary to return 'state'. Switch statement doesn't require a default return as function will return 'state' at the end regardless.

Reply:
https://github.com/mate-academy/js_stateful-object/pull/2537/files#r1252276239
not fixed this comment. it means that no need to return state. better to throw some error for example.

Reply:
I think i'm not 100% understood this issue
should I just in 'default' case to throw 'new Error' object and not to return state both from switch statement and function at all?